### PR TITLE
Updates a bunch of tips and adds new ones

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Updated a bunch of roundstart tips and added new ones.
/:cl:

A lot of tips were pretty outdated (nanofrost sprays? telescience?) and of course, all the recent stuff like techweb point generation, assmos and chef CQC were missing.
I added a few tips as well, let me know if I got anything wrong.

<details><summary>Click here for wall of text</summary>
<p>
- As a Chemist, you can make bottles from plastic sheets. Great for when you run out of beakers! <br>
- As a Scientist, you can use the mutation toxin obtained from green slimes to turn yourself into a jelly mutant. Each subspecies has unique features - for example telepathic powers, duplicating bodies or integrating slime extracts! <br>
- As a Scientist, you can generate research points by letting the tachyon-doppler array record increasingly large explosions.  <br>
- As a Scientist, getting drunk just enough will speed up research. Skol! <br>
- As a Roboticist, you can reset a cyborg's module by pulsing the reset wire with a multitool. <br>
- As the Chief Engineer, your locker contains a jetpack upgrade that can be attached to hardsuits. <br>
- As an Engineer, you can cool the supermatter crystal by spraying it with a fire extinguisher. Only for the brave! <br>
- As an Engineer, you can convert tesla coils into corona analyzers by using a screwdriver on the circuit board. They will produce less power, but generate research points! <br>
- As an Engineer, you can use radiation collectors to generate research points. Load them with a 50/50 oxygen/tritium tank and use a multitool to switch them to research mode. <br>
- As an Atmospheric Technician, your backpack firefighter tank can launch resin. This resin will extinguish fires and replace any gases with a safe, room-temperature airmix. <br>
- As an Atmospheric Technician, your ATMOS holofan projector blocks gases while allowing objects to pass through. With it, you can quickly contain gas spills, fires and hull breaches. Or, use it to seal a plasmaman cloning room. <br>
- As an Atmospheric Technician, burning a plasma/oxygen mix inside the incinerator will not only produce power, but also gases such as tritium and water vapor. <br>
- As the Clown, you can use your stamp on a sheet of cardboard as the first step of making a honkbot. Fun for the whole crew! <br>
- As a Cook, being in the kitchen will make you remember the basics of Close Quarters Cooking. <br>
- As a Shaft Miner, you can craft a variety of equipment from the local fauna. Bone axes, lava boats and ash drake armour are just a few of them! <br>
- You can swap floor tiles by holding a crowbar in one hand and a stack of tiles in the other.
</p>
</details>

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -31,19 +31,21 @@ As a Medical Doctor, you can surgically implant or extract things from people's
...
-As a Scientist, you can inject yourself with the mutation toxin extracted from green slimes to become one of many races, including a slimeperson, who will never be attacked by slimes, can split and even regenerate limbs!
+As a Scientist, you can use the mutation toxin obtained from green slimes to turn yourself into a jelly mutant. Each subspecies has unique features - for example telepathic powers, duplicating bodies or integrating slime extracts!
 As a Scientist, you can maximize the number of uses you get out of a slime by feeding it slime steroid, created from purple slimes, while alive. You can then apply extract enhancer, created from cerulean slimes, on each extract.
-As a Scientist, you can disable anomalies by scanning them with an analyzer, then send a signal on the frequency it gives you with a remote signalling device. This will leave behind an anomaly core, which is good for research or the construction of a Phazon mech!
+As a Scientist, you can disable anomalies by scanning them with an analyzer, then send a signal on the frequency it gives you with a remote signalling device. This will leave behind an anomaly core, which is good for research!
```
Comment (SpaceManiac) [inline strings/tips.txt]: You removed the correct information and left the incorrect here. Anomaly cores do nothing for research but can still be used to build Phazon mechs.

File: strings/tips.txt [hunk 2]
```diff
@@ -31,19 +31,21 @@ As a Medical Doctor, you can surgically implant or extract things from people's
...
 As a Scientist, researchable machine parts can seriously improve the efficiency and speed of machines around the station. In some cases, it can even unlock new functions.
-As a Scientist, with high enough bluespace research, you can build machines that can teleport objects and people all around the station! Look into setting up a telescience department if you can.
-As a Roboticist, keep an ear out for anomaly announcements. If you get your hands on an anomaly core, you can build a Phazon mech!
```
Comment (SpaceManiac) [inline strings/tips.txt]: Same

**Comment by Xhuis** [comment]
`- You can swap floor tiles by holding a crowbar in one hand and a stack of tiles in the other.`
WHAT.

**Comment by davethwave** [comment]
- As a Roboticist, you can reset a cyborg's module by pulsing the reset wire with a multitool.
It should be this since that was removed
- As a Roboticist, you can reset a cyborg's module by cutting and mending the reset wire with a wire cutter.

**Comment by MrDoomBringer** [comment]
- When hacking doors, cutting and mending the "test light wire" will send power back into the door.

**Comment by Tacolizard** [comment]
there should be a system for adding and removing tips based on a special tag in a pr changelog. that way you could easily add tips and remove them if the pr gets reverted.

**Comment by Shadowflame909** [comment]
- Service borgs get there own stun baton! Although, it's easily blockable with a gas mask.

**Comment by 81Denton** [comment]
@Tacolizard I fully agree.

I'll add more lines once I get back from work.

**Comment by 81Denton** [comment]
Are the cult/nukie tips still good? I've been playing lowpop for way too long.

**Comment by davethwave** [comment]
The cult ones seem good but I didn't check any of the nukie ones

**Comment by SpaceManiac** [comment]
>that way you could easily add tips and remove them if the pr gets reverted.

Editing the tips file is enough for this, reverting the PR will revert the tips file edits

some guy on IRC pointed out that "upgrade the grab by clicking on the grab button in your active hand" needs to be fixed

**Comment by 81Denton** [comment]
@SpaceManiac Alright, it's fixed.

**Pull request merged**